tests/profiling/collector/test_threading.py

```diff
@@ -72,7 +72,7 @@ def test_lock_acquire_events():
     # It's called through pytest so I'm sure it's gonna be that long, right?
     assert len(event.frames) > 3
     assert event.nframes > 3
-    assert event.frames[0] == (__file__, 65, "test_lock_acquire_events")
+    assert event.frames[0] == (__file__.replace(".pyc", ".py"), 65, "test_lock_acquire_events")
```

Comment:
related?

Reply:
Probably related to now having a `ModuleWatchdog` installed.

Reply:
this might be a good argument for moving enabling `ModuleWatchdog.install()` by default it's own PR, since these things seem un-related to the gevent change.